packages/main/src/components/workspace-language/WorkspaceLanguage.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import axios from "axios";
+
+const myStyle = {
+  inputfield: {
+    margin: "10px 0",
+    width: "70%"
+  },
+  btn: {
+    backgroundColor: "#00B87C",
+    color: "#ffffff",
+    border: "none"
+  }
+};
+
+const modifyOrganizationSettings = () => {
+  let organizationid = "6158c1ac3a1e49b1e5165df3";
+  let workspacelanguage = "";
```

Comment:
Same here. If you have it hardcoded anywhere else in the admin settings pages please fetch it dynamically. I believe it's usually stored in localStorage

Reply:
Request changed and updated